backend/src/email/utils/callback/parsers/ses.ts

```diff
@@ -222,6 +224,7 @@ const parseRecord = async (record: SesRecord): Promise<void> => {
   await blacklistIfNeeded(message)
 
   const id = smtpApiHeader?.unique_args?.message_id
+  const isTransactional = smtpApiHeader?.isTransactional
```

Comment:
```suggestion
  const isTransactional = !!smtpApiHeader?.isTransactional
```

Reply:
just a nit, so it won't be `undefined` (type-safetyytyyzz)

Reply:
undefined is evaluated to falsy in js, it's common to not do `!!`